src/Symfony/Component/ErrorHandler/Resources/views/exception_full.html.php

```diff
@@ -22,13 +22,6 @@
                             <span class="hidden-xs-down">Symfony</span> Docs
                         </a>
                     </div>
-
-                    <div class="help-link">
-                        <a href="https://symfony.com/support">
-                            <span class="icon"><?= $this->include('assets/images/icon-support.svg'); ?></span>
```

Comment:
Can we remove the icon-support.svg file or is it used elsewhere?

Reply:
I can't find any other occurrence, so I've removed the icon file too.